test/tests/pbs_smoketest.py

```diff
@@ -562,12 +562,11 @@ def test_submit_job_with_script(self):
         """
         Test to submit job with job script
         """
-        a = {ATTR_rescavail + '.ncpus': '2'}
-        self.server.manager(MGR_CMD_SET, NODE, a, id=self.mom.shortname)
         j = Job(TEST_USER, attrs={ATTR_N: 'test'})
-        j.create_script('sleep 120\n', hostname=self.server.client)
+        j.create_script('sleep 5\n', hostname=self.server.client)
         jid = self.server.submit(j)
         self.server.expect(JOB, {'job_state': 'R'}, id=jid)
+        self.server.expect(JOB, 'queue', op=UNSET, id=jid, offset=5)
         self.logger.info("Testing script with extension")
         j = Job(TEST_USER)
         fn = self.du.create_temp_file(suffix=".scr", body="/bin/sleep 10",
```

Comment:
This isn't part of your change, but I don't understand the next part of this test.  We don't submit the job with -N, so why is it OK for the job submit to fail as long as we don't fail with a -N failure message?  Shouldn't it be a test failure if the job fails to submit?  If you want to leave it alone, that is OK.

Reply:
I Agree with you @bhroam . It doesn't make any sense to catch qsub error and checking error message here. Test should fail if qsub fails. I have removed try-except block. Please have a look.